parity/blockchain.rs

```diff
@@ -406,25 +404,30 @@ fn execute_import(cmd: ImportBlockchain) -> Result<(), String> {
 
 	service.register_io_handler(informant).map_err(|_| "Unable to register informant handler".to_owned())?;
 
+	let timer = Instant::now();
 	client.import_blocks(instream, cmd.format)?;
+	let elapsed = timer.elapsed();
+	let ms = timer.elapsed().as_millis();
 
 	// save user defaults
 	user_defaults.pruning = algorithm;
 	user_defaults.tracing = tracing;
 	user_defaults.fat_db = fat_db;
 	user_defaults.save(&user_defaults_path)?;
 
+	// Sometimes when importing a small number of blocks (~10) the report seems
+	// inaccurate, so we sleep a little before getting the report.
+	std::thread::sleep(Duration::from_secs(1));
```

Comment:
huh?

Reply:
yeah I have no idea wtf. If you try importing 5-10 blocks the report is all messed up, claiming wrong number of blocks were imported. I tossed in this and moved on without investigating further.

Reply:
I was hoping we could figure this out before the merge.

Reply:
Firstly apologies for adding this change that has nothing to do with the rest of the PR. I'll remove it.

Secondly, I was not overly worried about it because the import is nondeterministic right? Some (user configurable) verifier threads are going to process the import queue and if the number is small maybe there's a race on the `ClientReport` so giving it a little more time, I thought, was "fine"?